jenkins/job-configs/kubernetes-jenkins-pull/bootstrap-maintenance-pull.yaml

```diff
@@ -44,7 +44,11 @@
         repo-name: 'k8s.io/test-infra'
     - janitor:
         branch: master
-        frequency: '0 */3 * * *'
+        frequency: 'H */3 * * *'
         job-name: maintenance-pull-janitor
         repo-name: 'k8s.io/test-infra'
-
+    - aws-janitor:
+        branch: master
+        frequency: 'H * * * *'
+        job-name: maintenance-aws-janitor
```

Comment:
The weird thing is that I'm not exactly sure where to put this. It's actually maintaining the whole AWS billing account (and could be extended to maintain multiple), but it looks like the top-level `bootstrap-maintenance.yaml` isn't intended for this. I'll go ahead and rename it for now, though.